packages/web/components/skeleton.tsx

```diff
@@ -0,0 +1,24 @@
+import classNames from "classnames";
+import { DetailedHTMLProps, FunctionComponent } from "react";
+
+const Skeleton: FunctionComponent<
```

Comment:
Could we name this something like `LoadingSkeletonPlaceholder` or something to clarify it's for loading data?

Reply:
Renamed it into `SkeletonLoader`